src/category_theory/abelian/transfer.lean

```diff
@@ -189,4 +198,159 @@ def abelian_of_equivalence
   (F : C ⥤ D) [functor.preserves_zero_morphisms F] [is_equivalence F] : abelian C :=
 abelian_of_adjunction F F.inv F.as_equivalence.unit_iso.symm F.as_equivalence.symm.to_adjunction
 
+section transfer_enough_injectives
+
+/-!
+If `L ⊣ R` are a pair of adjoint functors between abelian categories `𝓐` and `𝓐` and `L` is
+faithful and exact, then if `𝓑` has enough injectives, so does `𝓐`. We achieve this by considering
+an arbitrary injective presentation of `L(A) ⟶ J`: by adjunction, there is an `A ⟶ R(J)`, we will
+prove that this `A ⟶ R(J)` is an injective presentation of `A`.
+-/
+
+open limits adjunction
+
+universes v₁ v₂
+
+variables {C : Type u₁} {D : Type u₂} [category.{v₁} C] [category.{v₂} D] [enough_injectives D]
+variables (L : C ⥤ D) (R : D ⥤ C)
+
+namespace enough_injectives_of_adjunction_auxs
+
+/--
+Given injective presentation `L(A) → J`, then `injective_object_of_adjunction A` is defined to be
+`R(J)`. It will later be proven to be an injective object in `𝓐`.-/
+def RJ (A : C) : C := R.obj $ injective.under (L.obj A)
+
+local notation `RJ_of` := RJ L R
+
+variables (adj : L ⊣ R)
+variables {L R}
+
+/--
+If `g : X → R(J)` and `f : X → Y` is mono in `𝓐`, then there is an morphism `L(Y) → J`
+See the diagram below:
+
```

Comment:
I think that if you add backticks, then the diagram will look better in mouse-over popups in VSCode.
````suggestion
```
````